scripts/toolboxSrc/applyEnvVarsToClientAssets.ts

```diff
@@ -55,7 +55,7 @@ const rewriteIndexHTML = () => {
     github: process.env.GITHUB_CLIENT_ID,
     google: process.env.GOOGLE_OAUTH_CLIENT_ID,
     googleAnalytics: process.env.GA_TRACKING_ID,
-    mattermostDisabled: process.env.MATTERMOST_DISABLED === 'true',
+    mattermostDisabled: !!process.env.MATTERMOST_SECRET || process.env.MATTERMOST_DISABLED === 'true',
```

Comment:
+1 i'm probably not thinking about this correctly.
if mattermost is disabled, how do they turn it on?

Reply:
This env is used to hide the row in the team integrations UI. With these variables set, the plugin will be available on the whole instance. Notification settings will be configured in Mattermost.
This version will not use the `sharedSecret` integration provider I've added in another PR. That would be a cleaner solution for the future, but I'll go a simpler route for now.